x-pack/plugins/alerting/common/execution_log_types.ts

```diff
@@ -36,7 +36,16 @@ export interface IExecutionLog {
   timed_out: boolean;
 }
 
+export interface IErrorLog {
+  id: string;
+  timestamp: string;
+  type: string;
+  message: string;
+}
+
 export interface IExecutionLogResult {
```

Comment:
Do you think it would be clearer to define a separate type?

```
export interface IErrorLogResult {
  totalErrors: number;
  errors: IErrorLog[];
}

export type IExecutionAndErrorLogResult = IExecutionLogResult & IErrorLogResult;
```